CHANGES.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 1.1.0
```

Comment:
```suggestion
## 2.0.0
```

Let's use the `2.0.0` major version as dropping the python3.6 support is pretty much a breaking change